subsquid/src/model/generated/asset.model.ts

```diff
@@ -27,6 +27,12 @@ export class Asset {
   @Column_("numeric", {transformer: marshal.bigintTransformer, nullable: false})
   price!: bigint
 
+  /**
+   * Defaults to 12 when not specified
```

Comment:
as was dicussed in slack with other people either no default or (that is mine) make 12 only last step decision with TODO item to remove that. do not provide decimals from data layer if it is not aware of decimals.

Reply:
OR add gh/cu item to fix data layer with no hardcode.

Reply:
> is title still corect? xcm does not propagates decimals nor allows to hook to convert, or any other known bridge. also unkown assets do. ot have decimals and statemine assets can be without. so decimals only ui side concern only. and usdt is 6 decimals.

@andresmechali yeah i believe you dont need to store normalized values, but one thing keep in mind are pablo transactions, when there are two assets involved (they might have a different decimals) we need to be sure in that case whether we will calculate correctly or not (e.g spot price)

Reply:
The issue is that if I don't normalize values, I cannot properly provide a value that makes sense for queries like `totalValueLocked`.

Reply:
@saadjhk regarding your comment about Pablo, we do treat assets independently so that's not an issue